simulation/tests/test_agents.py

```diff
@@ -580,8 +584,12 @@ def test_total_heating_system_costs_are_lower_for_heat_pumps_if_model_interventi
             intervention="boiler_upgrade_scheme",
         )
 
-        assert household.get_total_heating_system_costs(
-            heat_pump, model_with_boiler_upgrade_scheme
-        ) < household.get_total_heating_system_costs(
-            heat_pump, model_without_boiler_upgrade_scheme
+        assert sum(
+            household.get_total_heating_system_costs(
+                heat_pump, model_with_boiler_upgrade_scheme
+            ).values()
```

Comment:
Note that if we changed the output to a tuple we could simply drop `.values()` in all of the tests